skrf/io/general.py

```diff
@@ -300,19 +303,27 @@ def read_all(dir='.', contains = None, f_unit = None, obj_type=None, files=None)
 
     out={}
 
-    filelist = files
-    if files == None:
-        filelist = sorted(os.listdir(dir))
+    if not os.path.isdir(dir):
+        dir = os.path.join(os.getcwd(), dir)
```

Comment:
if `dir` is not a directory, why `join(os.getcwd(), dir)` would be a directory? Do you have an example ?

Reply:
I originally had a reason for this, but I cannot remember. After further testing, it seems unnecessary to be there.

Reply:
This must have been something I did in my test configuration where I forgot to correct the current path.